src/main/app/controller/PostController.ts

```diff
@@ -63,6 +69,20 @@ export class PostController<T extends AnyObject> {
       res.redirect(nextUrl);
     });
   }
+
+  private getAnsweredUnreachableResponsesAsNull(req: AppRequest<T>): Partial<Case> {
```

Comment:
stop shoving everything in the post controller :D

Reply:
👍 Fixed in 07670ab.